src/lesson_8/script.js

```diff
@@ -0,0 +1,260 @@
+function arrayCreate () 
+{
+    const arrayLength = Number(prompt('яка довжина масива?'));
+    if(arrayLength > 0) {
+        let element;
+        const newArray = [];
+
+        for(let i = 1; i <= arrayLength; i++) {
+            element = prompt('what should be inside this element?');
+            newArray.push(element);
+        }
+
+        return newArray;
+    } else {
+        console.log('wrong parameter :(');
+    }
+}
+
+const someNumbers = [4, 5, 10, -3, 9, 1, 24];
+someNumbers.sort(function(num_1, num_2) {
+    return num_1 - num_2;
+});
+
+const shoppingList = [
+    {
+        productName: 'bread',
+        productPrice: 14.5,
+        productQuantity: 2
+    },
+    {
+        productName: 'milk',
+        productPrice: 25,
+        productQuantity: 2
+    },
+    {
+        productName: 'cucumber',
+        productPrice: 7,
+        productQuantity: 25
+    },
+    {
+        productName: 'oil',
+        productPrice: 32,
+        productQuantity: 4
+    },
+    {
+        productName: 'salt',
+        productPrice: 13,
+        productQuantity: 8
+    },
+    {
+        productName: 'meat',
+        productPrice: 92.8,
+        productQuantity: 1
+    },
+];
+
+const calculatorPrice = shoppingList.reduce(function(prev, cur) {
+    return prev + cur.productPrice;
+  }, 0);
+
+function getMinQuantity(shoppingList)
+{
+    if( Array.isArray(shoppingList) && shoppingList.length > 0) {
+       
+        let min = shoppingList[0].productQuantity;
+        for (let i = 0; i < shoppingList.length; i++) {
+
+            if(shoppingList[i].productQuantity < min) {
+                min = shoppingList[i].productQuantity;
+            }
+        }
+
+        return min;
+    }
+}
+
+const numberProducts = shoppingList.reduce(function(prev, cur) {
+    return prev + cur.productQuantity;
+  }, 0);
+
+function getMostExpensive(shoppingList)
+{
+    if( Array.isArray(shoppingList) && shoppingList.length > 0) {
+       
+        let max = shoppingList[0].productPrice;
+        for (let i = 0; i < shoppingList.length; i++) {
+
+            if(shoppingList[i].productPrice > max) {
+                max = shoppingList[i].productPrice;
+            }
+        }
+
+        return max;
+    }
+}
+
+function addItem(shoppingList) 
+{
```

Comment:
Please, lint your code

```suggestion
function addItem(shoppingList)  {
```